javascript/javascript1/week4/voiceassistant.js

```diff
@@ -0,0 +1,54 @@
+let toDoList = [];
+
+function getToday() {
+  let today = new Date();
+  let day = today.getDate();
+  let month = today.getMonth() + 1;
+  let year = today.getFullYear();
+  let marr = new Array(
+    "January",
+    "February",
+    "March",
+    "April",
+    "May",
+    "June",
+    "July",
+    "August",
+    "September",
+    "October",
+    "November",
+    "December"
+  );
+  console.log(day + " of " + marr[month] + " " + year);
+}
+
+function myFunction() {
+  console.log("Timer Done");
+}
+
+function getReply(command) {
+  if (command == "Hello my name is Benjamin") {
+    console.log("nice to meet you Benjamin");
+  } else if (command == "Add fishing to my todo") {
+    toDoList.push("fishing");
+  } else if (command == "Add singing in the shower to my todo") {
+    toDoList.push("singing in the shower");
+  } else if (command == "Remove fishing from my todo") {
+    console.log("removed fishing from todo");
```

Comment:
Try to use different string functions like startsWith(), endsWith() to check fist and last matching words.

In this case if string starts with "remove" and ends with "todo", then you should remove item from list or array.